user_data/strategies/Strategy002.py

```diff
@@ -11,18 +11,18 @@
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 import numpy # noqa
 from datetime import datetime
+import threading
 
-from wao.util import execute, back_test
+from wao._429_file_util import create_429_directory, setup_429
 from wao.config import Config
-from user_data.strategies.notifier import send_start_deliminator_message
 from wao.strategy_controller import StrategyController
 
 
 class Strategy002(IStrategy):
     controller = StrategyController()
+    controller.send_starter_message('Freq Strategy002 ')
```

Comment:
pass this param in the constructor and call the **private** method send_starter_message in the constructor